api/openapi-spec/swagger.json

```diff
@@ -51920,7 +51920,7 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.Status"
+              "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta",
```

Comment:
this file is generated... the type registered for Writes/Returns has to be fixed in staging/src/k8s.io/apiserver/pkg/endpoints/installer.go for the DELETE and DELETECOLLECTION cases

then, run `hack/update-openapi-spec.sh` to regenerate this file

also, this isn't just an issue for the jobs API, it affects all delete/deletecollection types